Zend/tests/match/013.phpt

```diff
@@ -0,0 +1,93 @@
+--TEST--
+Match expression string jump table
```

Comment:
Would be useful to have a similar test of opt_debug_level with ZEND_MATCH_LONG (possibly one with an explicit default arm?)